### ISSUE TITLE
Build error

### ISSUE THREAD

**Issue by jricc** [trishullab/Quasimodo#3]
Hello,
When I compile Quasimodo with the command "invoke build-quasimodo" I get the error :
/usr/bin/ld: final link edit failed: wrong value
collect2: error: ld returned 1 exit status
make : *** [Makefile:43 : libquasimodo.so] Error 1
Could you help me?

**Comment by cs14b052** [comment]
Did all the previous steps in README/install.sh work as expected?

**Comment by jricc** [comment]
Yes, I tried again this morning with the same result. Do you have a Docker image for Quasimodo?

**Comment by cs14b052** [comment]
Sorry for the late reply. Here's the DOI - 10.5281/zenodo.7922448. However, a lot has changed since then. If things didn't work out still, can you please send an email to the email-id listed on the paper - https://arxiv.org/pdf/2302.04349.pdf. Thanks!
